sites/partners/src/settings/ManageIconSection.tsx

```diff
@@ -32,7 +32,12 @@ const ManageIconSection = (props: IconContentProps) => {
           />
         </button>
         {props.onDelete && (
-          <button onClick={props.onDelete} aria-label={"Delete"} data-test-id={props.deleteTestId}>
+          <button
+            onClick={props.onDelete}
+            aria-label={"Delete"}
+            data-test-id={props.deleteTestId}
```

Comment:
Why do we have both test ids here?

Reply:
This was an issue I wasn't able to figure out how to fix. React testing library by default uses the `data-testid` format whereas our cypress tests look for `data-test-id`. We were able to get around this previously by adding `configure({ testIdAttribute: "data-test-id" })` to setup-tests.js. That would make it so react testing library would look for the same format as cypress and we didn't need to add new test ids to our code. However, with one of the package updates it is no longer working and when you run the tests it is again looking for `data-testid`. I wouldn't mind looking into the issue more, but I switched this one spot for now to get around the issue for now

Reply:
I was able to figure out the issue. shared-helpers and partners were pulling in different versions of react testing library which was causing conflict. I've updated them to be the same and it's working again so I was able to delete the second test id